content/docs/user-guide/experiment-management/visualizing-plots.md

```diff
@@ -15,22 +15,29 @@ workflow is:
    ...
    ```
 
-2. [Define plots](#defining-plots), optionally using
+2. [Define plots](#defining-plots) in `dvc.yaml`, optionally using
    [templates](#plot-templates-data-series-only) to configure how to visualize
    the data.
 
    ```yaml
    plots:
-     evaluation/test/plots/confusion_matrix.json: # Configure template and axes.
+
+     # Data series source
+     evaluation/test/plots/confusion_matrix.json:
+       # Configure template and axes.
        template: confusion
        x: actual
```

Comment:
Currently it's not clear that it's 3 plots and what each one represents:

<img width="755" alt="image" src="https://user-images.githubusercontent.com/1477535/197957266-84f77625-527f-492a-84af-3c6181ce8958.png">

Reply:
The need for code comments may be a sign of an overcomplicated sample though, maybe we should focus on a single type up here (happy path) and have longer examples of all of them further down?